core/lib/config/src/configs/chain.rs

```diff
@@ -45,6 +45,18 @@ impl Default for FeeModelVersion {
     }
 }
 
+#[derive(Debug, Clone, Copy, Deserialize, PartialEq, Eq)]
+pub enum L1BatchCommitDataGeneratorMode {
+    Rollup,
+    Validium,
+}
+
+impl Default for L1BatchCommitDataGeneratorMode {
```

Comment:
Nit: You can derive `Default` and add `#[default]` attribute for the `Rollup` variant in order to be more concise and idiomatic.

Reply:
Addressed in [34f9f37](https://github.com/matter-labs/zksync-era/pull/1015/commits/34f9f37e3bb212277fad6af3fee04f6e6c6f2db2)